ctd.py

```diff
@@ -16,7 +16,7 @@ class CTD(sensor.Sensor):
     def __init__(
             self,
             data_dir=f'/home/{config.munkholmen_user}/olmo/munkholmen/DATA',
-            file_regex_l0=r"ready_ctd_(\d{14})\.csv",
+            file_search_l0=r"ready_ctd_(\d{14})\.csv",
```

Comment:
Same as previous comment about considering adding a comment to explain explicitly what L0 data in this case refers to?

Reply:
See above.